src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,57 @@
+package duke.main;
+
+import duke.command.Command;
+import duke.errors.DukeException;
+import duke.task.TaskList;
+
+/**
+ * Represents the Duke programme. Carries out the commands and handles exceptions
+ */
+public class Duke {
+    private static final String FILEPATH = "data/duke.txt";
```

Comment:
I like how you made this a constant so that it will be easier to change in the future.